Makefile

```diff
@@ -527,3 +527,23 @@ kubebuilder:
 	mkdir -p ${ENVTEST_ASSETS_DIR}
 	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.7.0/hack/setup-envtest.sh
 	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR);
+
+MANIFESTS := webhook/all_in_one_$(API_VERSION).yaml
+all-in-one-build-webhook: ## Re-generate all-in-one yaml
+	@echo "+ $@"
+	> $(MANIFESTS)
+	cat webhook/rbac.yaml >> $(MANIFESTS)
+	cat webhook/operator.yaml >> $(MANIFESTS)
+	cat webhook/webhook.yaml >> $(MANIFESTS)
+	cat webhook/cert-manager.yaml >> $(MANIFESTS)
+	sed -i "s~{DOCKER_REGISTRY}:{GITCOMMIT}~${DOCKER_REGISTRY}:${GITCOMMIT}~;" ${MANIFESTS} 
+	
+
+setup-cluster: minikube-start
+	@echo "+ $@"
+	eval $(minikube docker-env)
+
+deploy-webhook: setup-cluster install-crds container-runtime-build all-in-one-build-webhook 
+	@echo "+ $@"
+	kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.4.0/cert-manager.yaml
+	kubectl apply -f ${MANIFESTS}
```

Comment:
Please add a TODO in line 530 to integrate it with master Makefile when you're done with the whole task :)

Reply:
Okay sure.